trajan/accessor.py

```diff
@@ -47,6 +48,40 @@ def __new__(cls, ds):
 
         tx = detect_tx_dim(ds)
 
+        # if we have a 1D dims, this is most likely some contiguous data
+        # there may be a few exceptions though, so be ready to default to the classical 2D parser below
+        if len(tx.dims) == 1:
+            # only support ContiguousRagged for now
+            ocls = ContiguousRagged
+            
+            # we have a dataset where data are stored in 1D array
+            if "index" in tx.dims:
+                obsdim = "index"
+
+                if "time" in ds.coords:
```

Comment:
I believe this need not be a variable named "time", but `the variable which has a standard_name attribute equal to "time"`

Reply:
Use cf-xarray to resolve that, but it must be a time variable which is a coordinate of a trajectory related variable. There could be other times (e.g. imu time).

Reply:
Yes, we could utliise cf-xarray more (also in OpenDrift), instead of duplicating functionality that exists there.

Reply:
Ah right. That's not covered by cf-xarray (which is why we have trajan :D): I think you have to look at the longitude variable, you figure out its dimension and then you find a time variable (standard name time) with the same dimension. I think that is pretty close to what you are doing, just use standard name rather than variable name.

Reply:
> @gauteh browsing a bit through cf-xarray; do you have some example / pointer to some other places in trajan from where I should get inspiration from? :)

I was thinking about this functionality: https://cf-xarray.readthedocs.io/en/latest/selecting.html#by-standard-name , then you can select by standard_name and role. I want to migrate to using this for selecting longitude and latitude variables as well.

Reply:
Yes, that is the main feature of the CF-conventions: variable (and dimension) names are ad-hoc, the attributes (e.g. standard_name) are the standardized keys to use.
However, TrajAn was also from the beginning based on assumption that dimensions should be named "trajectory" and "obs", but this is also false - and just based on random examples in CF documentation.
Thus some general cleanup is needed "one day".

But yes, generic parsing is difficult, and it is quite unclear what is the best practice, as discussed here:
https://github.com/OpenDrift/trajan/discussions/123

Reply:
I believe time can never be a coordinate (i.e. `time(time)`) for ragged arrays, so it wil always be a normal variable with `standard_name="time"` (i.e. `time(obs)` as in CF-examples, or `time(index)` as in this case).

But this time variable should also have the same dimension as `lon and lat`, although we should ideally also allow for `X and Y` in case of another CRS :-)

Reply:
I think X and Y is already accepted here ? 😊

Reply:
Yes, I think so.